### DISCUSSION TITLE
Ability to increase the size of fonts for Menu Bar, Tabs, Column Names & Status Bar

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1119] [Suggestions]
### Description

Was wondering if it would be considered to add the ability to be able to increase the size of the fonts used for....
Menu Bar
Tabs
Column Names
Status Bar

Menu Bar might be a part of the Linux desktop environment itself but if not (for example in the standalone version) this would be nice to have.  (Often when fonts settings are handled in a desktop environments they tend to group many items together, so you may like the font size of one element but not another)
  
Can add the ability to do this in the "Fonts" section in "Options" like the others.
Also a cool feature that DC provides to achieve this is to be able to hold Ctrl+Mouse Scroll to increase / decrease font size, which I love.
Have seen other file managers offer this in file panes but never in items such as the address bar which DC does, very nice touch.
If too complex to set this up for all the items mentioned, still would be nice to be able to change the fonts sizes in the Options – Fonts section.
 
I think this covers most of the text/font based elements in DC that are part of the main GUI.
This would be great if could be added so users can match the font sizes of these items to the items that can already do this.
 
Thank You

**Comment by Skif-off** [comment]
_Menu Bar / Tabs:_ These are standard components, appearance depends on the system theme.
_Status Bar:_
DC 1.0.x: I'm not sure if this was added to the 1.0.x branch.
DC 1.1.0 (next release or current alpha): go to Configurations in the main menu > Options... > Fonts >Status bar.

P.S. I don't know about column headers. Theoretically, this is part of the file list (i.e. the same graphic interface component) and the size depends on the font/icon size of the file list.

**Comment by advice2020** [comment]
Thank you for your response, very happy to hear about the Status Bar font, I just tried out the 1.1.0 AppImage, it currently does not have the Ctrl+MSW zoom capability but as mentioned the main thing is at least having this in the Options – Fonts section.
 
MENU BAR
Unfortunately the Standalone and AppImage versions do not update the font size when these are changed in the DE, at least not for me anyways.
 
TABS
You mention may also be part of the DE as well?, I did not know that.  If this is true then that means that those fonts will probably also not update.
 
COLUMN NAMES
For me these fonts do not change either.  I use Ctrl+MSW on file pane to increase the size of font which works for files but not column headers.  Which in this case it is good because I personally make my file list font rather large and would not want the “Column Names” font to be the same size.
Hopefully these two fonts are not linked to same setting.
 
Probably not something that is going to be addressed but would their be some way of being able to override the font settings used by your OS / DE where these can be added to the Options – Fonts and used in just DC like all the other font settings?  This way these fonts could be changed in all versions of DC.
 
Eventually I will try an installed version if I could get the Alpha version to work, and see what fonts updated when they are changed within my DE, if I find out anything useful to share I will post here.
Just unfortunate because plan was to be able to use the stand alone version of DC.
 
If anyone has any more info to share please do so.

**Comment by Skif-off** [comment]
You can try to change your GTK2 theme (see [this](https://doublecmd.h1n.ru/viewtopic.php?t=7867) attempt). Using the `GTK2_RC_FILES` environment variable, you can try to apply changes only to DC.
I guess there is something similar for Qt, but I don't know.
